engines/tinsel/pcode.cpp

```diff
@@ -274,7 +274,7 @@ static INT_CONTEXT *AllocateInterpretContext(GSORT gsort) {
 			pic->GSort = gsort;
 			return pic;
 		}
-#ifdef DEBUG
+#ifdef DEBUGA
```

Comment:
This looks to be wrong

Reply:
Correct, that shouldn't be in there. Fixed